src/tests/frontend/controller/test_sqlite_controller.py

```diff
@@ -142,24 +331,53 @@ def testValidatePluginNameIfOk(self):
   def testValidatePluginNameIfNotOk(self):
     """test the validate plugin name method if not ok"""
     with tempfile.TemporaryDirectory() as tmpdir:
-      file = os.path.join(tmpdir, 'testfile')
-      pathlib.Path(file).touch()
+      path = os.path.join(tmpdir, 'testfile')
+      pathlib.Path(path).touch()
 
       output_handler = output_handler_file.OutputHandlerFile(
-          file, file_handler.FileHandler())
+          path, file_handler.FileHandler())
       plugin_helper = fake_sqlite_plugin_helper.FakeSQLitePluginHelper(
           valid_name=False, change_bool_after_every_call_valid_name=True)
       controller = sqlite_controller.SQLiteController(output_handler,
                                                       plugin_helper)
       controller._ValidatePluginName("the_wrong_plugin_")
       expected = 'Plugin is not in a valide format. Choose new name [' \
                  'plugin_name_...]: '
-      actual = self._ReadFromFile(file)
+      actual = self._ReadFromFile(path)
+      self.assertEqual(expected, actual)
+
+  def testCreateSQLQueryModelWithUserInput(self):
+    """test the creation of the sql query model with the user input"""
+    query = "select x"
+    with_examples = True
+    query_execution = fake_sqlite_query_execution.SQLQueryExecution(
+        base_sql_query_execution.SQLQueryData(
+            data=['first', 'second', 'third', 'fourth']))
+
+    with tempfile.TemporaryDirectory() as tmpdir:
+      path = os.path.join(tmpdir, 'testfile')
+      pathlib.Path(path).touch()
+
+      output_handler = output_handler_file.OutputHandlerFile(
+          path, file_handler.FileHandler())
+      plugin_helper = fake_sqlite_plugin_helper.FakeSQLitePluginHelper()
+      controller = sqlite_controller.SQLiteController(output_handler,
+                                                      plugin_helper)
+      controller._CreateSQLQueryModelWithUserInput(
+          query, with_examples, query_execution
+      )
+      expected = 'Your query output could look like this.\n' \
```

Comment:
Instead of trailing backslashes use:
```
      expected = (
          'Your query output could look like this.\n'
          'first\n'
          'second\n'
          'third'
          'Do you want to add this query?'
          'What kind of row does the SQL query parse?')